README.md

```diff
@@ -128,10 +157,14 @@ $ curl -F "image=@assets/dog-human.jpg" -XPOST http://127.0.0.1:5000/model/predi
 The optional `threshold` parameter is the minimum `probability` value for predicted labels returned by the model.
 The default value for `threshold` is `0.7`.
 
-## 4. Development
+### 4. Development
 
 To run the Flask API app in debug mode, edit `config.py` to set `DEBUG = True` under the application settings. You will then need to rebuild the docker image (see [step 1](#1-build-the-model)).
 
+### 5. Cleanup
```

Comment:
This needs to become "Clean up" so that the link at https://github.com/IBM/MAX-Object-Detector/pull/19/files#diff-04c6e90faac2675aa89e2176d2eec7d8R71 works

Reply:
Nice catch, that must've been a copy-paste error. I updated the link about to match, Cleanup is the noun version which makes more sense in this context than the verb "clean up" (also fixed on skeleton)

Reply:
Did you actually push a change? I can't see an update

Reply:
the command was sitting in a terminal tab un-entered >.<

Reply:
I suspected as much :)

Reply:
hmm in other places we have gone with Clean Up rather than Cleanup. So it would be inconsistent now. We should be consistent across repos of course. 

It doesn't matter much either way but I'd say that the majority are probably currently "Clean Up" ...

Reply:
Since I'll be following this up with updates to every model I can make sure to update them. From a grammar perspective using the noun as a title makes more sense, if we were to use the verb version it should be Cleaning Up rather than Clean Up and would still require a change. (yay English grammar)